tests/dispatch/test_http2.py

```diff
@@ -65,6 +79,20 @@ def test_http2_post_request():
     }
 
 
+async def test_http2_large_async_post_request(backend):
```

Comment:
Can we group these two tests together, like other sync/async pairs? Also the convention in this file seems to be test_async_... for the async test cases. :)

Reply:
Changed order and name in new commit.